libraries/AP_Airspeed/AP_Airspeed_I2C.cpp

```diff
@@ -22,27 +22,41 @@
 
 #include <AP_Common/AP_Common.h>
 #include <AP_HAL/AP_HAL.h>
+#include <AP_HAL/I2CDevice.h>
 #include <AP_Math/AP_Math.h>
+#include <stdio.h>
+#include <utility>
 
 extern const AP_HAL::HAL &hal;
 
 #define I2C_ADDRESS_MS4525DO 0x28
 
+#if CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_DISCO
+#define I2C_BUS_MS4525DO HAL_BARO_MS4515DO_I2C_BUS
+#else
+#define I2C_BUS_MS4525DO 0
```

Comment:
Should be 1. See lucasdemarchi/ardupilot/commit/db9a1d7c644ecff24f36fedfa11fbb4285ada29f